pytorch_toolkit/object_detection/model_templates/requirements.txt

```diff
@@ -1 +1,2 @@
 ./packages/ote
+nncf==1.4.1
```

Comment:
Agree that it may be a wrong level.

What do you think if I add the line
```
-r ../../external/mmdetection/requirements/nncf_compression.txt
```
to the file `pytorch_toolkit/object_detection/requirements.txt`?

Note that at the moment it will make the same -- install `nncf==1.4.1`

Reply:
Done.
Moved it to the file `pytorch_toolkit/object_detection/requirements.txt`

2 @druzhkov-paul : please, write me if this is not a proper `requirements.txt` file for NNCF dependency in your opinion